.gitignore

```diff
@@ -4,3 +4,4 @@ build
 dist
 *.egg-info
 *.egg
+.idea
```

Comment:
No harm with this, but I've recently realized it's much easier to put this kind of tool-specific stuff into a global git ignore on each dev machine I'm working on. e.g., put ".idea" into ```~/.gitignore``` and then

```
git config --global core.excludesfile ~/.gitignore
```